src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -59,76 +57,39 @@ protected CuDisbursementVoucherTaxService getCuDisbursementVoucherTaxService(){
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankAddress());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankRoutingNumber());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankAccountNumber());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankSwiftCode());
+    protected boolean hasWireTransferValues(final PaymentSourceWireTransfer wireTransfer) {
+        boolean hasValues = super.hasWireTransferValues(wireTransfer);
+        final PaymentSourceWireTransferExtendedAttribute wireExtension =
+                (PaymentSourceWireTransferExtendedAttribute) wireTransfer.getExtension();
+        hasValues |= StringUtils.isNotBlank(wireExtension.getBankStreetAddress());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getBankProvince());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getBankSWIFTCode());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getBankIBAN());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getSortOrTransitCode());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getCorrespondentBankName());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getCorrespondentBankAddress());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getCorrespondentBankRoutingNumber());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getCorrespondentBankAccountNumber());
+        hasValues |= StringUtils.isNotBlank(wireExtension.getCorrespondentBankSwiftCode());
         return hasValues;
     }
 
     @Override
-    protected void clearWireTransferValues(final PaymentSourceWireTransfer dvWireTransfer) {
-        super.clearWireTransferValues(dvWireTransfer);
-        final DisbursementVoucherWireTransferExtendedAttribute wireExtension =
-                (DisbursementVoucherWireTransferExtendedAttribute) dvWireTransfer.getExtension();
-        wireExtension.setDisbVchrBankStreetAddress(null);
-        wireExtension.setDisbVchrBankProvince(null);
-        wireExtension.setDisbVchrBankSWIFTCode(null);
-        wireExtension.setDisbVchrBankIBAN(null);
-        wireExtension.setDisbVchrSortOrTransitCode(null);
-        wireExtension.setDisbVchrCorrespondentBankName(null);
-        wireExtension.setDisbVchrCorrespondentBankAddress(null);
-        wireExtension.setDisbVchrCorrespondentBankRoutingNumber(null);
-        wireExtension.setDisbVchrCorrespondentBankAccountNumber(null);
-        wireExtension.setDisbVchrCorrespondentBankSwiftCode(null);
+    protected void clearWireTransferValues(final PaymentSourceWireTransfer wireTransfer) {
+        super.clearWireTransferValues(wireTransfer);
+        final PaymentSourceWireTransferExtendedAttribute wireExtension =
+                (PaymentSourceWireTransferExtendedAttribute) wireTransfer.getExtension();
+        wireExtension.setBankStreetAddress(null);
```

Comment:
Another area that could be simplified by moving the logic into a new `PaymentSourcePreRulesService` implementation.